python/setup.py

```diff
@@ -114,7 +115,7 @@
     "scipy",
 ]
 
-extras["streaming"] = ["msgpack >= 0.6.2"]
+extras["streaming"] = ["msgpack >= 1.0.0"]
```

Comment:
Can we factor this out into a variable or something now that it seems we have to keep it in sync?

Reply:
Actually, I can just remove this one (empty list, so ray[streaming] works) , and it'll be in one place